doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -190,6 +190,7 @@ The following table indicates the software maturity levels of the support for ea
 .. toggle::
 
   .. sml-table:: bluetooth
+     :insert-values: [("Bluetooth LE Peripheral/Central","nRF54H20","Experimental"), ("Connectionless/Connected CTE Transmitter","nRF54H20","Experimental"), ("LE Coded PHY","nRF54H20","Experimental")]
```

Comment:
@FrancescoSer  shouldn't it be removed? I think if changes nRF54H20 column content that LLPM is stated as not available.

Reply:
No, LLPM  is stated as not available even without this line. This line just changes what was there to experimental. Default is supported. What's the correct info there? Is it available?